src/tap/__init__.py

```diff
@@ -1,5 +1,10 @@
+"""
+Typed Argument Parser
+"""
+
+__version__ = "1.10.0"
```

Comment:
In case you wanna merge this PR, it probably makes sense to bump this to `1.10.1`

Reply:
That makes sense! Thank you for the reminder!